vendor/woocommerce/action-scheduler/classes/WP_CLI/Action_Command.php

```
@@ -0,0 +1,353 @@
+	 * <id>
+	 * : The ID of the action to get.
+	 * ---
+	 * default: 0
+	 * ---
+	 *
+	 * @param array $args Positional arguments.
+	 * @return void
+	 */
+	public function logs( array $args ) {
+		$command = sprintf( 'action-scheduler action get %d --field=log_entries', $args[0] );
+		WP_CLI::runcommand( $command );
```

PHPStan (2.1.46)

[error] Call to static method runcommand() on an unknown class Action_Scheduler\WP_CLI\WP_CLI.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)